benchmarks/sidekiq_test.rb

```diff
@@ -1,4 +1,5 @@
 ENV['RAILS_ENV'] = 'production'
+require 'English'
```

Comment:
What's this? This doesn't look right.

Reply:
http://ruby-doc.org/stdlib-2.3.0/libdoc/English/rdoc/English.html

Before the cleanup it was required by one of dependencies - 'ruby-prof' probably. But after removing that require it stopped working.

Reply:
Whoa, I'd never seen that before. If it's a part of the standard lib, I guess it's okay.